docs/MAINTAINERS.adoc

```diff
@@ -322,36 +322,26 @@ Code reviews do NOT pursue the following goals:
 
 == Issue triage
 
-Jenkins core and most of its components use link:https://issues.jenkins.io/[Jenkins Jira] as an issue tracker.
-This issue tracker is open to all Jenkins users.
-They report defects and requests for enhancements,
-and then component maintainers triage issues and provide feedback to users.
-In the case of the Jenkins core, the *Issue Triage Team* and *Core Maintainers* are roles that are expected to process the incoming issues.
+The *Issue Triage Team* and *Core Maintainers* are roles that are expected to process the incoming issues.
```

Comment:
Thoughts about just using the mention of the roles?

Reply:
what do you mean?